ampersand-select-view.js

```diff
@@ -47,9 +47,7 @@ module.exports = View.extend({
         }
 
         this.el = opts.el;
-        if (opts.label === undefined) {
-            this.label = this.name;
-        } else {
+        if (opts.label !== undefined) {
```

Comment:
We can set it as `this.label = opts.label || '';` and get rid of `render` check. It should work the same.